src/Value/ValueAbstract.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace StadGent\Services\OpeningHours\Value;
+
+/**
+ * ValueAbstract.
+ *
+ * @package StadGent\Services\OpeningHours\Value
+ */
+abstract class ValueAbstract implements ValueInterface
+{
+    /**
+     * @inheritdoc
+     */
+    public function sameValueTypeAs(ValueInterface $object)
```

Comment:
ipv "sameValueTypeAs" eventueel "sameValue" in de ValueInterface definieren;
Vervolgens in deze abstract klasse de type check doen;
Vervolgens in de kind klassen parent::sameValue() oproepen?